bundles/org.openhab.binding.playstation/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:playstation:PS3">
+		<parameter name="ipAddress" type="text" required="true">
+			<context>network-address</context>
+			<label>IP address</label>
```

Comment:
if both are allowed, better use `host` or `hostname` instead of `ipaddress`

Reply:
I changed the description to only ipAddress since I can't test with hostname in my network.